core/src/main/java/org/jdbi/v3/core/statement/PreparedBatch.java

```diff
@@ -204,11 +204,7 @@ private <R> R execute(ResultProducer<R> producer, Consumer<ExecutedBatch> batchC
                 return batch.stmt;
             }, getContext());
         } catch (SQLException e) {
-            try {
-                close();
-            } catch (Exception e1) {
-                e.addSuppressed(e1);
-            }
+            Cleanable.cleanUpForException(e, this);
```

Comment:
I noticed this works slightly different.  Originally it added the exception thrown by `close` as surpressed, the new code (in `cleanUpForException` adds the cause of the exception (it may not have any!).